README.md

```diff
@@ -880,3 +881,10 @@ If you do like our project and we hope that you do, can you please support us? O
 - Your company's logo can be featured on **Readme** and **Website**
 - Intended for medium-sized companies
 
+## Supporters
+Parts of infrastructure for this project is supported by:</p>
```

Comment:
My suggestion:

`Some parts of the infrastructure for this project are supported by:`